packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -366,6 +366,18 @@ class ServiceConnectionManager {
     );
   }
 
+  Future<Response> sendDwdsEvent(
+      {@required String screen, @required String action}) async {
+    return await _callServiceExtensionOnMainIsolate(registrations.dwdsSendEvent,
```

Comment:
hide this behind `if (kIsWeb)` since this should not be send for the desktop devtools

Reply:
done!